ManualTests/image-resampling-with-scale.html

```diff
@@ -0,0 +1,33 @@
b15/mjG3dxdX8+SYOShY5lsqA4Oz7H1Jyah0AgkDKSpkifQQonTxWGjF88f9cnrr36PlLRuhZR0JBbtd/3vIT9FT1dDA+fZ3LyMpVySBBkFtDONQZ8+bMfYM+OHs5PlHj61XEuTIUUKxqEoL3Jp73JYyDfzFXdLYzPhAyNFhadEQF4UuBLgZSOjC982tvX7AceAErxb9O6ELd86d7tnb2bvxYEAVIKt0CFIAiynDs3grUWrQ1eimBzY8Df3flhBnvWcejoGC+8OcmlmZCKMslaKRQ1Y1NlTgzPMna5zODGZla3BIzPhFiclQNfks1IGqLD90SEIZKGlIBYO7B55yNPHfr3sUQyKQJeZ9+2mxE1eRmwPr+GfL7TAdKGsFJ1Bl/69PvZsKaNHzz7S06en3ULVhuUMoTKEIbuHH8+NFLg4Z+OIIVgIN9Ee1OGjlyGXINH1pd4cun0bHDnjpvThk8TyARNTdfWehxwmhwc7HceAVCholyqsHtbF7u3dfHYi6OMTJTQ2vn4MAavNKEyqPgcfT86WeLQS6O0NvjkGlaajrlZaGzKXQtk6hLIeNltUsi6vm1Vewvd3V1J2hCGipt2b+bMhTlOvDPtwCuDWmB9nZBRSqOUC2JvDs9ydmyOtsalHUMtgYZs47bFCPgyyG4QUkYZ5cK2Y/tmstks1lryq1q4aiDPCycuOclER2L9UM+TUEwqJqi04djpywS+W6zLNYtTghcEG0g5nzQBGYXMBQlx3IKMx+DgJgA6V+ew1jJ0ftZJIwKp1Hyrh0o7MikiMdnTIwWsZVndJwyqnQR3ev5EEtLjcF+nPOnv6+LMmXfYOZDHWpieC/E9iTGRy4v8uRDz742lF0dubSwXp8pYLIEvKCuWbalMOBnar/cDYy3SCndR45I8T7Jjx1asLWOspaJd+uBLlx95ifutPiWa2KgWIEk1lHbp80pKr1gVtaFjHoG42JBWRMmWwKszu/nODgoz4xgLUgoqyqClQIr4ICERP90SgU/NQCbjMs6KWroMriaAUVK5GIH5BYpIzrVxQQpBQ/sarLX0djbx2tkppHDWl5F8JMyL2PHYxpIkfP0bc+56mTI+zmId+fnfpQnYSrk4mg0yeSmFKyisQFqbpMnpVjGCsZmQbT2tHB2aTMDLVP6ywDjMl9C121YT6qicXBK8u6+iNLOzM6dIqS7thUy5MHNSKY01UWFiqjl7vfbGSJHdgx1ckW+iFGrKSlNSmlJY51CaUmiSfm9nE/0bchTKK5GPs77ShkJhbgTQ9QiomctjLyqta4pvu0B3cXvrYomRqQr7P9aP54kIrKGoNMUEuFlAZl17ljs/2k9FGSpqCeunZWcgDDVjFy68uBiBcOjo04fSZWDtUa8dPjlNMbTc88c76cs311jdgS+mPuvLN/PlWwaREqaLi1s/0bwlKYxCY/mvZx4/RKoySyvVBzpvP/j8T1pa2weygY/nSTxP4EmXZHlRelvbAk+wd3Mr61ozPPvqGEeGJnnp1ETVBVvLezd18N6BVVy3fS0jlysYA03ZRYqkiEBcwYXaUiwrLl6aGPr967dcD1wAVAw6bhqYu3D6te81bP+dv/J9DyEsArevmYZdS6KiLU+fmKJvTZYdfavYu33tPDDxv4mC4rk3p7AWdnTllgQfS1fFBCohJ19/5XvAHCkJ1ZqzAei+/b6fPZfL5TqDwHfWT82CTIqMhZ4mbs1Z6YqYRmfhsakKkwXFbFnjS8EHt7bXtX6y25EEOkslsv7E5ekLf7Bv015gmFRBU5s7V4DJ0VO/+FYlVMlWR5zLJ4V5sglVf10UyoZzkxVeO1/k+C/nGJ4oM1t2RtuyvnEB+LSfj/WutLN8RVnmyiGv/feRbwGTEcak1ZrBAvbkz3789tZ9n7lO+kHe87xEywvgxvE9qsjrzUh6ryeX9djV0zJPguk9pjhGxLKpKMvMXImL4xePfeUPP3I3cImarZV6q8gAtlwunNq49f2fEEIE1SJnYXSKNW6xKb1Xf6tSacLVvTnamvzUPVU36axPFby2FIoVZovlwg8OfmP/228cOwkUau1Yj4AF1PjZ12faNgxMr8r3fcgiEBGJ2GLxtEM1T8Ha1O6dW4ShdrXx+raAzfkmIMqzIEorXGrh3CSE2lBWlrmSYraseOGZx7/+8Df/9gmcfHQt2MXqOQuoMy8/9c6a7i2llnXde8AlaGkr25QVsfOBxxsAxlg8IbjmihayvoysHbvIyMdrCLUlVNaBL4fMlkJeeu7pe+7/2h3fBS5So/3lCBCxDd868sSpju4txZa1PXuMtdGe5fztwqR+sGCTzVlQxhG4Ym0DGzsanLUjmShDtFCdG64oS0lZZosVZkuKo889ec/Bv/zcgzifX4SFS3A5AhYXLMqnjzxxqmVN18WWdb3vM8YGMiru06E+vcUeXyttyPqSnb2tgHCgUxavREc5Aj9dKDE9Vy688NR/HHjgr7/4r8AITveLhuzltgQSEmd/8czp2fHRn6/u27lV+JlO64rUFPCUfAzu5YWxbMo309acSXYkEuCRXEqhoVAKmZ6rMDY2dvzhg1//88e/849PRpZfEjys/B2ZxAW5VUDnDXf+w+9t3PK+W4PGpnUZ38dFbVdCxoWMtZaWRp9r+tsR8cJNeRttLKVQUSor5gqFsRNHf/rdb//N578fAZ/EBatlX/i9q5d8uNSjBVgNrP7gHX//qQ2brvlktrm1TwiB50mklNGOsmVHbxstjZnEzyulqzsXxjI7ffntt175+Y++c+ALj+B8/CVgBjfrK6k0f7XXrEAWaMbNSHvve24cGPzAx/e1rOve1dCQ68w0tfa1NWfo6cxByvrF2am3C4XpC5fOnT125KlHDr9y+LEh4DLO4gWgzP/ha9Z6RAKgEWgCclG/AbfxVJumGFwULeG8yiwuMSviXORv7EV3vTE8HOhM1PfqjG1xrlnjiIRRf0VS+X/b/geyPrDn1efsgAAAAABJRU5ErkJggg==";
```

Comment:
Can't we write a ref test instead? For example, can't we compare this test with drawing the image to a canvas?

Reply:
Now will try this after checking above changes whether they work in EWS or not. :-)